### ISSUE TITLE
[Feature Request]: GLSL & HLSL

### ISSUE THREAD

**Issue by timdevorg** [Azn9/JetBrains-Discord-Integration#123]
### Describe the feature you'd like

GLSL Files are partly supported, the files are titled “OpenGL” but the icons are not present for the classic, material and flat theme.

![Screenshot 2024-06-11 at 13 45 08](https://github.com/Azn9/JetBrains-Discord-Integration/assets/42376598/9c1b5f56-fcc7-4311-b7f0-ee01cdac49b0)

The atom theme, however, has an icon

![Screenshot 2024-06-11 at 13 44 18](https://github.com/Azn9/JetBrains-Discord-Integration/assets/42376598/55bdb8bf-7603-46f6-9672-fff94311bda6)

---

TLDR: I'd love to see some unity across these themes when it comes to GLSL and HLSL Shader language files. This is not really a big problem, though, so it can be done whenever there is time for it.

Khronos provides the logos on their website:
https://www.khronos.org/legal/trademarks/

**Comment by Azn9** [comment]
Added with https://github.com/Azn9/JetBrains-Discord-Integration/pull/131 ;)
